test/common/stats/thread_local_store_test.cc

```diff
@@ -90,6 +91,121 @@ class StatsThreadLocalStoreTest : public testing::Test, public RawStatDataAlloca
   std::unique_ptr<ThreadLocalStoreImpl> store_;
 };
 
+class HistogramTest : public testing::Test, public RawStatDataAllocator {
+public:
+  typedef std::map<std::string, Stats::ParentHistogramSharedPtr> NameHistogramMap;
+  InSequence s;
```

Comment:
nit: move the InSequence declaration down to bottom of the class declaration with all the other member vas.